common/v2/features/BroadcastTransaction/components/ConfirmTransaction.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { useState, useContext } from 'react';
 import styled from 'styled-components';
 import BN from 'bn.js';
+import { Transaction } from 'ethereumjs-tx';
+import { addHexPrefix } from 'ethereumjs-util';
```

Comment:
Is this ConfirmTransaction component specific to Broadcast Transaction flow? Can it be folded in using the one from `v2/components/TransactionFlow` ?

If not, do the changes made in this Confirm component need to also be implemented in `v2/components/TransactionFlow/ConfirmTransaction` ?

Reply:
It was made before the state factory pattern was introduced; it is a wrapper with some logic around `ConfirmTransaction` from TransactionFlow. I can probably move some functions to the `BroadcastFlow` file and use pure `ConfirmTransaction` component.

Reply:
I reused the `GeneralStepper` component, moved the logic from custom `ConfirmTransaction` component to `stateFactory` and get rid of it.